inc/taxonomies.php

```diff
@@ -62,16 +62,6 @@ function largo_custom_taxonomies() {
 
 	$termsDefinitions = array(
 		array(
-			'name' => __( 'Sidebar Featured Widget', 'largo' ),
```

Comment:
Just a quick sanity check on this...so, if a site already has these terms created, they'll still exist, right? This would probably be the way we'd want to go so that if they're using the recent posts widget to query stories from this particular prominence term it won't break.